packages/react/src/patterns/blocks/ContentBlockSimple/README.md

```diff
@@ -0,0 +1,103 @@
+# Content Block - Simple
+
+> The Content Block - Simple pattern is a decorator of `ContentBlock`, and
+> includes a single `ContentItem`, optional media (image), and ends with a CTA.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-simple/content-block-simple';
+```
+
+> 💡 Only import fonts once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { SimpleLongForm } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <ContentBlockSimple
+      heading={heading}
+      copy={copy}
+      items={items}
+      mediaData={mediaData}
+      mediaType={mediaType}
+      cta={cta}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the simplelongform styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name        | Required | Data Type | Default Value | Description                                                              |
+| ----------- | -------- | --------- | ------------- | ------------------------------------------------------------------------ |
+| `heading`   | YES      | String    | n/a           | Title of the content block.                                              |
+| `copy`      | YES      | String    | n/a           | Introduction copy briefly describing the section.                        |
+| `items`     | YES      | Array     | n/a           | Array of content items for use as content body.                          |
+| `mediaData` | NO       | Array     | n/a           | Array of image sizes for use with content items.                         |
+| `mediaType` | NO       | String    | 'image'       | Type of media. Currently only supports `image`.                          |
+| `cta`       | NO       | Object    | n/a           | CTA used at the end of content body. `Text` and `Card` styles supported. |
```

Comment:
Can we link to the CTA documentation?

Reply:
Sure. I think it's a good idea to go back through all the READMEs and do this.

Reply:
Trying to catch them as we go.